2Dgame/Assets/Scripts/Platform Scripts/BlinkerScript.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BlinkerScript : PlatformScript {
+
+    new Renderer renderer;
+
+    private new void Awake()
+    {
+        renderer = GetComponent<Renderer>();
+    }
+
+    /**
+     * Enables the renderer when jumped on
```

Comment:
This doesn't seem to happen? Is the idea for them to be shown again when you jump on them again? When I've played they stay hidden.